utils/lit/lit/TestingConfig.py

```diff
@@ -16,9 +42,16 @@ def fromdefaults(litConfig):
         Create a TestingConfig object with default values.
         """
         # Set the environment based on the command line arguments.
+
+        # strip dxil validator dir if not need it.
+        all_path = os.pathsep.join(litConfig.path +
+                                     [os.environ.get('PATH','')])
+        need_dxil_validator = litConfig.params.get('need_dxil_validator', None)
+        if need_dxil_validator == None:
+            all_path = strip_dxil_validator_path(all_path)
```

Comment:
See previous comment about `need_dxil_validator`.  Do we rely on the path to look up to find the TAEF binaries we are testing?  If not, we should just make sure the DXIL.dll we want to test is next to that binary, and not ever picked up from the path.  If we do, then I'm worried that we need to change the path stripping code, or add the path to the test content after stripping any paths leading to potentially unwanted DXIL.dlls.

Reply:
Removed need_dxil_validator.